lib/kubeclient.rb

```diff
@@ -358,6 +357,12 @@ def create_faraday_client(url = nil)
         if @auth_options[:username] && @auth_options[:password]
           connection.basic_auth(@auth_options[:username], @auth_options[:password])
         end
+
+        refresh_bearer_token_from_file
+        if auth = @headers[:Authorization]
+          req.headers['Authorization'] = auth
```

Comment:
this is kinda sus ... did not see where this was supposed to be coming from before